capi/VideoDecoderCapi.h

```diff
@@ -19,16 +19,10 @@
  *  Boston, MA 02110-1301 USA
  */
 
-#ifndef __VIDEO_DECODER_CAPI_H__
-#define __VIDEO_DECODER_CAPI_H__
+#ifndef videodecodercapi_h
+#define videodecodercapi_h
```

Comment:
Yes , we should use webkit style guards. but some code from codec parser use capital letters as guards. It's not gone be problem any more. 
But we do not need change it in this patch. Hi @clearlylin , could we only change need line?